### PR TITLE
Updated default help installation path

### LINKED ISSUE #304: Helper generate wazuh splunk app outdated
Hello team.

Help message from wazuh-packages/splunkapp/generate_wazuh_splunk_app.sh is out of date
```
 -s, --store <directory> [Optional] Destination directory by default /tmp/splunk-app
```

New path is
```
wazuh-packages/splunkapp/output/
```

Regards.
Alejandro Alguacil

### PR DESCRIPTION
**Pull request by alguacilaguamara**
Hello @BraulioV.

To close #304 I update text help wazuh-packages/splunkapp/generate_wazuh_splunk_app.sh in https://github.com/wazuh/wazuh-packages/commit/3d44baf7cbcc6103f0521dbbe7bb9102fcb85290 and README.md in https://github.com/wazuh/wazuh-packages/pull/305/commits/716e1d7c89152f343365eea9b3ee6ee9d925295a.

Regards.
Alejandro Alguacil

### CODE REVIEW

File: splunkapp/generate_wazuh_splunk_app.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ help() {
     echo "Usage: $0 [OPTIONS]"
     echo
     echo "    -b, --branch <branch>     [Required] Select Git branch or tag e.g. 3.8 or v3.8.1-7.2.3"
-    echo "    -s, --store <directory>   [Optional] Destination directory by default /tmp/splunk-app"
+    echo "    -s, --store <directory>   [Optional] Destination directory by default splunkapp/output/"
```
Comment (BraulioV) [inline splunkapp/generate_wazuh_splunk_app.sh]: I would change this to `${CURRENT_PATH}/output`. This variable is defined here: https://github.com/wazuh/wazuh-packages/blob/716e1d7c89152f343365eea9b3ee6ee9d925295a/splunkapp/generate_wazuh_splunk_app.sh#L16 and is used to get where is located the `generate_wazuh_splunk_app.sh`

File: wazuhapp/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ To build a Kibana app package, you need to download this repository and use the
     Usage: ./generate_wazuh_app.sh [OPTIONS]
 
         -b, --branch <branch>     [Required] Select Git branch or tag e.g. 3.8-6.7 or v3.7.2-6.5.4
-        -s, --store <path>        [Optional] Set the destination path of package, by defauly /tmp/wazuh-app.
+        -s, --store <path>        [Optional] Set the destination path of package, by defauly splunkapp/output/
```
Comment (BraulioV) [inline wazuhapp/README.md]: In this case, I would change it to `/path/to/generate_wazuh_app.sh` to remove any hard-coded path that may confuse the user.

File: splunkapp/generate_wazuh_splunk_app.sh (hunk 1 above)
Comment (alguacilaguamara) [reply splunkapp/generate_wazuh_splunk_app.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/305/commits/1dedde988334f5d9ba81861915474ca4e99b6c10

File: wazuhapp/README.md (hunk 2 above)
Comment (alguacilaguamara) [reply wazuhapp/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/305/commits/1dedde988334f5d9ba81861915474ca4e99b6c10

**Pull request merged**